### PR TITLE
Simplify using-gatsby-image example #11329

### PR DESCRIPTION
**Pull request by lstar19**
## Description

Write a more explicit walkthrough for using gatsby-image with GraphQL.

## Related Issues
Addresses #11329

### CODE REVIEW

**Comment by sidharthachatterjee** [comment]
Great work @lstar19 👍 

I'd love to hear what @marcysutton has to say about this

File: docs/docs/using-gatsby-image.md [hunk 1]
```diff
@@ -37,25 +37,56 @@ With Gatsby, we can make images way way better.
 
 `gatsby-image` is designed to work seamlessly with Gatsby’s native image processing capabilities powered by GraphQL and Sharp. To produce perfect images, you only need to:
 
-1. Import gatsby-image and use it in place of the built-in img
-1. Write a GraphQL query using one of the included GraphQL “fragments” which specify the fields needed by `gatsby-image`.
+1. Install `gatsby-image` and its peer dependencies. The `gatsby-config.json` needs to include the following:
+
+```js:title=src/gatsby-config.js
```
Comment (LekoArts) [inline docs/docs/using-gatsby-image.md]: The config file is in the root, not in src :)

File: docs/docs/using-gatsby-image.md [hunk 2]
```diff
@@ -37,25 +37,56 @@ With Gatsby, we can make images way way better.
...
+plugins: [
```
Comment (LekoArts) [inline docs/docs/using-gatsby-image.md]: Would be cool if you write out the complete form of the config file, not only the plugins array (not that people paste that into the file only)

File: docs/docs/using-gatsby-image.md (hunk 1 above)
Comment (lstar19) [reply docs/docs/using-gatsby-image.md]: Might have messed up the connections between the PRs, but updated the code to address this and below comment: https://github.com/gatsbyjs/gatsby/pull/11998

File: docs/docs/using-gatsby-image.md (hunk 2 above)
Comment (lstar19) [reply docs/docs/using-gatsby-image.md]: #11998

**Comment by sidharthachatterjee** [comment]
Closing this in favour of https://github.com/gatsbyjs/gatsby/pull/11998

**Pull request closed without merging**